ui/package.json

```diff
@@ -37,6 +38,7 @@
     "change-case": "^3.1.0",
     "classnames": "^2.2.6",
     "codemirror": "^5.39.0",
+    "core-js": "3",
```

Comment:
Why are we adding this dependency?

Reply:
Reason for core js is during yarn start, it gives you are using corejs 2.6 please upgrade to atleast 3 and build stalls for 3-4 sec at the same warning.

Reply:
Without its presence the build gets stall on the netify as well as on terminal . Its due to the updated `React-Scripts`

Reply:
Which `React-Scripts`? Do you have a link I can look at or something. Just trying to understand :)